Lombiq.HelpfulExtensions/Lombiq.HelpfulExtensions.csproj

```diff
@@ -51,4 +57,14 @@
     <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="5.1.1" />
```

Comment:
Also put `Node.Js` here as a NuGet package under the same condition as HL. Example: https://github.com/Lombiq/Orchard-Chart.js/blob/dev/Lombiq.ChartJs/Lombiq.ChartJs.csproj#L46